src/olympia/activity/utils.py

```diff
@@ -33,6 +33,9 @@
 ACTIVITY_MAIL_GROUP = 'Activity Mail CC'
 NOTIFICATIONS_FROM_EMAIL = 'notifications@%s' % settings.INBOUND_EMAIL_DOMAIN
 SOCKETLABS_SPAM_THRESHOLD = 10.0
+# Types of users who might be sending or receiving emails.
+ADDON_AUTHOR = 'An add-on author'
```

Comment:
I find it a little confusing that it's both a constant to identify what type of user it is and the string that will be used as the sender name... but only if they are a reviewer. I think I would rather have different constants: `ADDON_AUTHOR` and `ADDON_REVIEWER` integers to denote the type of user, then another would like `ADDON_REVIEWER_NAME` or something for the name string sent to developers.

What do you think?

Reply:
I also changed the consts to be `USER_TYPE_ADDON_AUTHOR` and `USER_TYPE_ADDON_REVIEWER` as that's more descriptive of why these consts exist.